### PR TITLE
Suhorutchenko hw1 t1 t2

### PR DESCRIPTION
**Pull request by mykoIa**
task completed

### CODE REVIEW

**Review by lember3 — DISMISSED** [review]

File: homework/english/sukhorutchenko/lecture1/vocabulary.md [hunk 1]
```diff
@@ -3,39 +3,59 @@
...
+The application is not yet able to execute this command
+
+to implement
+It is necessary to implement the interface for everything to work
+
+to install
+Install a new library 
+
+to plug in
+It is always necessary to plug in git it is convenient
```
Comment (Yur4eg1) [inline homework/english/sukhorutchenko/lecture1/vocabulary.md:45]: You have to divide the sentenсes for right structure:
It is always necessary to plug in _a_ git .It is convenient

File: homework/english/sukhorutchenko/lecture1/vocabulary.md [hunk 2]
```diff
@@ -3,39 +3,59 @@
...
- to uninstall
- 
- to update
- 
- to upgrade
- 
- to verify
- 
+to enable
+Enable a new fuch today
```
Comment (Yur4eg1) [inline homework/english/sukhorutchenko/lecture1/vocabulary.md:33]: fuch -?

**Review by Yur4eg1 — CHANGES_REQUESTED** [review]
see notes

**Review by mykoIa — COMMENTED** [review]
string 45 and 33

File: homework/english/sukhorutchenko/lecture1/vocabulary.md (hunk 1 above)
Comment (mykoIa) [reply homework/english/sukhorutchenko/lecture1/vocabulary.md]: ok

File: homework/english/sukhorutchenko/lecture1/vocabulary.md (hunk 2 above)
Comment (mykoIa) [reply homework/english/sukhorutchenko/lecture1/vocabulary.md]: changed

**Review by Eliseyhom — DISMISSED** [review]
lgfm

File: homework/english/sukhorutchenko/lecture1/grammar.md [hunk 3]
```diff
@@ -2,48 +2,48 @@
...
-3. At the moment Peter is work in Russia. 
-4. Does he got a new car?  
-5. He never wear a hat. 
-6. He don’t like black coffee. 
-7. We are have a good time.  
-8. What you doing now? 
-9. It rains at the moment.
-10. How you like the game?
+1. We do not go to school today
+2. What do you do after school?
```
Comment (goldenas18) [inline homework/english/sukhorutchenko/lecture1/grammar.md:32]: We are not going to school today. 2. What are you doing after school?

File: homework/english/sukhorutchenko/lecture1/vocabulary.md [hunk 4]
```diff
@@ -3,39 +3,59 @@
...
+We need to develop a bot telegram by the end of the month
+
+to disable 
+We need to disable this feature
 
 to disconnect
+Database to disconnect from server
 
 to display
+It doesn't display the screen
```
Comment (goldenas18) [inline homework/english/sukhorutchenko/lecture1/vocabulary.md:30]: maybe "on the screen"?

**Review by mykoIa — COMMENTED** [review]
grammar: str 31, 32
vocabulary: 30

File: homework/english/sukhorutchenko/lecture1/vocabulary.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+I optimize the server for fast boot
+
+to create
+I am creating a new project
+
+to connect
+I can not connect my project to the database
+
+to debug
+Today I'm doing a debugging program
```
Comment (goldenas18) [inline homework/english/sukhorutchenko/lecture1/vocabulary.md:15]: "Today I'm going to debug the program" or "Today I'm debugging the program"

File: homework/english/sukhorutchenko/lecture1/grammar.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+12. Look! The boys are coming home.
+13. Does Maila like Chinese food?
+14. My grandmother does not work. She is on pension.
+15. What is your brother watching now?
+16. Where are you living?
+17. Do you play the piano well?
+18. Where is their brother swiming now?
+19. What do they eat at breakfast?
+20. Are they swiming at the moment?
+21. Mike does not wear a tie today.
```
Comment (goldenas18) [inline homework/english/sukhorutchenko/lecture1/grammar.md:25]: isn't wearing

**Review by goldenas18 — APPROVED** [review]

**Review by mykoIa — COMMENTED** [review]
grammar: str 25
vocabulary: 15

**Review by goldenas18 — APPROVED** [review]
seems great)

**Review by Yur4eg1 — APPROVED** [review]

**Review by lember3 — APPROVED** [review]

**Pull request merged**